vetiver/tests/test_pytorch.py

```diff
@@ -100,38 +99,28 @@ def test_torch_predict_ptype_error():
     assert response.status_code == 422, response.text  # value is not a valid float
 
 
-def test_torch_predict_no_ptype():
+def test_torch_predict_no_ptype_batch():
     torch.manual_seed(3)
     x_train, torch_model = _build_torch_v()
-    v = VetiverModel(torch_model, model_name = "torch", save_ptype=False, ptype_data=x_train)
+    v = VetiverModel(torch_model, model_name = "torch")
     v_api = VetiverAPI(v, check_ptype=False)
 
     client = TestClient(v_api.app)
-    data = "3.3"
+    data =[[3.3], [3.3]]
```

Comment:
users will pass in matrices, rather than strings that need to be parsed. this makes more sense, imo, for a non-checked implementation anyway